src/modules/leaderboard/leaderboard.command.ts

```diff
@@ -94,35 +114,81 @@ export const LeaderboardCommand: Command<ApplicationCommandType.ChatInput> = {
 			const all = await DDUser.findAll();
 			await Promise.all(all.map(getActualDailyStreak));
 		}
-		const { format, value, name } = traitInfo;
+		const { format, value, sortingValue, select, name, useAggregation } =
```

Comment:
could this big block of code be moved into a separate function please, preferably with a bit of documentation